ios/FluentUI/PersonaButton/MSFPersonaButton.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+/// UIKit wrapper that exposes the SwiftUI PersonaButton implementation
+@objc open class MSFPersonaButton: NSObject, FluentUIWindowProvider {
+
+    /// The UIView representing the PersonaButton.
+    @objc open var view: UIView {
+        return hostingController.view
+    }
+
+    /// The object that groups properties that allow control over the PersonaButton appearance.
+    @objc open var state: MSFPersonaButtonState {
+        return personaButton.state
+    }
+
+    @objc public init(size: MSFPersonaButtonSize = .large,
+                      theme: FluentUIStyle? = nil) {
```

Comment:
As we discussed this morning, it's probably better to keep the convenience initializer that omits the theme parameter in the interest of making the objc calls less verbose.